scripts/mw-kube-agent-install-v2.sh

```diff
@@ -153,15 +165,14 @@ for file in "$MW_KUBE_AGENT_HOME"/*.yaml; do
       -e "s|MW_API_URL_FOR_CONFIG_CHECK_VALUE|$MW_API_URL_FOR_CONFIG_CHECK|g" \
       -e "s|MW_CONFIG_CHECK_INTERVAL_VALUE|$MW_CONFIG_CHECK_INTERVAL|g" \
       -e "s|MW_VERSION_VALUE|$MW_VERSION|g" \
-    "$file" |kubectl apply -f - --kubeconfig "${MW_KUBECONFIG}"
+    "$MW_KUBE_AGENT_HOME/$file" |kubectl apply -f - --kubeconfig "${MW_KUBECONFIG}"
+
+    # If the current file is the job, wait for it to complete
+    if [ "$file" = "update-configmap-job.yaml" ]; then
+      printf "\nFetching the lastest settings for your account ..."
```

Comment:
typo. It should be `latest` instead of `lastest`

Reply:
Made the correction